shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-handler/src/test/java/org/apache/shardingsphere/sharding/distsql/update/AlterShardingTableRuleStatementUpdaterTest.java

```diff
@@ -93,21 +94,21 @@ public void assertUpdate() throws DistSQLException {
         updater.checkSQLStatement(shardingSphereMetaData, statement, currentRuleConfiguration);
         ShardingRuleConfiguration toBeAlteredRuleConfiguration = updater.buildToBeAlteredRuleConfiguration(statement);
         updater.updateCurrentRuleConfiguration(currentRuleConfiguration, toBeAlteredRuleConfiguration);
-        Assert.assertEquals(1, currentRuleConfiguration.getTables().size());
+        Assert.assertThat(currentRuleConfiguration.getTables().size(), is(1));
```

Comment:
Consider using static import here and below.

Reply:
thanks, I will use another pr to modify